sql/2021/structured-data/classic_microformats_types.sql

```diff
@@ -0,0 +1,36 @@
+# standardSQL
+  # Count Classic Microformats types
+CREATE TEMP FUNCTION
+  getClassicMicroformatsTypes(rendered STRING)
+  RETURNS ARRAY<STRUCT<name STRING, count NUMERIC>>
+  LANGUAGE js AS r"""
+  try {
+    rendered = JSON.parse(rendered);
+    return rendered.microformats_classic_types.map(microformats_classic_type => ({name: microformats_classic_type.name, count: microformats_classic_type.count}));
+  } catch (e) {
+    return [];
+  }
+""";
+WITH
+  rendered_data AS (
+  SELECT
+    getClassicMicroformatsTypes(rendered) AS classicMicroformatsTypes
```

Comment:
(throughout) prefer underscore naming convention for SQL field names
```suggestion
    getClassicMicroformatsTypes(rendered) AS classic_microformats_types
```